wgsl/index.bs

```diff
@@ -6857,6 +6852,20 @@ That's not a full user-defined function declaration.
     [=Component-wise=] when |T| is a vector.
     (GLSLstd450Fma)
 
+  <tr algorithm="frem">
+    <td>|T| is [FLOATING]
+    <td class="nowrap">`frem(`|e1|`:` |T| `, `|e2|`:` |T| `) -> ` |T|
+    <td>Returns the floating point remainder of |e1| divided by |e2|, where sign of non-zero result matches sign of |e2|.
+    [=Component-wise=] when |T| is a vector.
+    (OpFMod)
```

Comment:
let's start a fund that would get a coin every time someone files a PR to fix this "typo" ;)

Reply:
I had the same concern, so I've added some comments below this making clear it's not a mistake, and pointing to the discussion above.
It won't stop people filing bugs about it, but it should stop someone getting as far as a PR.